data/sql_updates/create_sched_e_aggregate_candidate.sql

```diff
@@ -9,8 +9,10 @@ select
     sum(exp_amt) as total,
     count(exp_amt) as count
 from sched_e
-where exp_amt is not null
-and (memo_cd != 'X' or memo_cd is null)
+where
+    exp_amt is not null and
+    filing_form in ('F3X', 'F5') and
```

Comment:
@paulclark2 are you saying: 
- we should be using the `F57 table` instead of the `sched_e` table
- we should be using `filing_form` and not `form_tp`